src/operation/decision_model.jl

```diff
@@ -239,10 +253,13 @@ function build_pre_step!(model::DecisionModel)
         populate_contributing_devices!(get_template(model), get_system(model))
         add_services_to_device_model!(get_template(model))
 
-        @info "Make Initial Conditions  Model"
-        build_initial_conditions!(model)
-        initialize!(model)
-
+        run_initialization_step = get_initialize_model(get_settings(model))
+        initialization_file = get_initialization_file(model)
+        if run_initialization_step
+            @info "Make Initial Conditions  Model"
+            build_initial_conditions!(model)
+            initialize!(model)
```

Comment:
@daniel-thom here we would do something like this: 

```suggestion
            initialize!(model)
            elseif !(isempty(initialization_file)
                 # De-serialize initialization file
             else
                 error()
             end
```